.github/workflows/update_pages.yml

```diff
@@ -0,0 +1,59 @@
+name: Update GitHub Pages
+
+on:
+  push:
+    branches:
+      - 'ontodoc'
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-*'
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Check out repository
+      uses: actions/checkout@v2
```

Comment:
Since you're checking out the repository on a different branch later, and are not directly using repository files until then (as far as I can see), this step might be removable? But I'm not completely sure. Maybe you could try to uncomment it and see if it then fails? And if not, it can be removed.

Reply:
This step is needed, but it could probably be moved further down to just before we call mkdoc.sh

Reply:
Ah of course. It's needed to run the scripts. Then no matter. Just leave it where it is 👍